src/AppBundle/Form/CommentType.php

```diff
@@ -54,9 +47,6 @@ public function configureOptions(OptionsResolver $resolver)
         ));
     }
 
-    /**
-     * @return string
-     */
     public function getName()
```

Comment:
To be more precise, it's inherited from [FormTypeInterface](https://github.com/symfony/Form/blob/2.7/FormTypeInterface.php), but anyway this docblock comment is useless.